data/plugins.json

```diff
@@ -34,12 +34,10 @@
   },
   "transform-es2015-destructuring": {
     "chrome": 51,
-    "edge": 14,
```

Comment:
![screen shot 2016-10-06 at 2 30 22 pm](https://cloud.githubusercontent.com/assets/588473/19165359/74cb9dbe-8bd1-11e6-82db-26dae6d8d47f.png)

Example of what I mean where it's not 100%